compiler/rustc_type_ir/src/lib.rs

```diff
@@ -273,6 +284,7 @@ impl fmt::Debug for Variance {
             Variance::Contravariant => "-",
             Variance::Invariant => "o",
             Variance::Bivariant => "*",
+            Variance::Errvariant(_) => "💥",
```

Comment:
Can we not use emojis in the compiler? It's really awkward and impossible to type.

Reply:
This reminds me of how awkward it was to use that 🔬 file extension in a-mir-formality. I think it's cute, but please let's don't 😓

Reply:
Oh right, we need to type it in diagnostic patterns